lib/katello/tasks/upgrades/2.4/import_package_groups.rake

```diff
@@ -0,0 +1,12 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '2.3' do
+      task :import_package_groups => ["environment"]  do
+
+        User.current = User.anonymous_api_admin
+        puts _("Importing Package Groups")
+        Katello::PackageGroup.import_all
```

Comment:
Eventually maybe, but right now I don't think so.  There are just too many issues around systems and consumers being deleted, i'd rather not reindex everything just to get a few content types